cmd/snappy/cmd_shell.go

```diff
@@ -73,6 +73,12 @@ Use "sudo snappy enable-classic" to enable it.`))
 		}
 
 		fmt.Println(i18n.G(`Entering classic dimension`))
+		fmt.Println(i18n.G(`
+
+The home directory is shared between snappy and the classic dimension.
+Run the "exit" command to leave the classic shell again. All files
+created will be preserved.
```

Comment:
I'd suggest this shorter version:
```
The home directory is shared between snappy and the classic dimension.
Run "exit" to leave the classic shell.
```
The rationale being that data preservation is a touchy topic. If we say it's preserved, I'd be tempted to expand with caveats, which are boring an most likely uninteresting for the typical user.